source/_posts/2020-11-12-android-300-release.markdown

```diff
@@ -0,0 +1,128 @@
+
+- 4 Keyguard sensors - Sensors that represent various states from the Keyguard API, such as if a device is currently locked or has a password setup. These sensors will update during the periodic 15 minute interval.
+- Last Notification sensor - A very powerful sensor that requires a special permission you must grant to read all notifications posted on the device. All attributes of the notification are provided as attributes for the sensor. You can think of this sensor as a great way to integrate any app that posts a notification to your device allowing you to automate around it. Personally I have been using it to help integrate apps that don't integrate with Home Assistant Core a food delivery app for example, you can tell when your food has been delivered and automate around it. This sensor will update as soon as a notification is posted.
+- 3 Sleep as Android sensors - Users who have the [Sleep as Android](https://play.google.com/store/apps/details?id=com.urbandroid.sleep&hl=en_US) app installed can have sensors added that will represent the state of various events from the app. One big benefit this has is that you can tell when an alarm is currently active, snoozed, dismissed or if you are sleeping. This helps solve a lot of issues users have been having around trying to automate the Next Alarm sensor, which is a timestamp sensor and usually requires some sort of [templating](https://community.home-assistant.io/t/how-to-use-next-alarm-sensor/221686). These sensors all update immediately upon a state change.
+- Last Update Trigger - A sensor whose state will represent the reason for the last update that was sent to your Home Assistant instance. This sensor will update anytime an update is sent to your Home Assistant instance.
+
+## Sensor Settings
+
+We didn't stop at just adding new sensors, we also made enhancements to the overall sensor experience. Starting from version 2.5.0 certain sensors have custom settings that can help with what updates actually get sent to your Home Assistant instance.
+
+
+<p class='img'>
+<img src='/images/blog/2020-11-05-android-300-release/location_sensor_settings.png' alt='Screenshot of location settings'></a>
+Screenshot of location settings.
+</p>
+
+- Next Alarm - This sensor has a setting for an allow list. This means that if you have an app that reports really odd timestamps as an actual alarm, you can now ignore them by telling the app which packages to send reports from. By default the list is blank. Tasker users are recommended to make use of this setting.
+- Last Notification - This sensor also has an allow list to let the user create a list of apps from which they want notification data from. By default all notifications are sent to your Home Assistant server. We highly recommend that you setup an allow list as soon as you can think of one to prevent a high amount of updates. You will be surprised by the amount of data that will show up in a short time period.
```

Comment:
```suggestion
- Last Notification - This sensor also has an allow list to let the user create a list of apps from which they want notification data from. By default all notifications are sent to your Home Assistant server. We highly recommend that you set up an allow list as soon as you can think of one to prevent a high amount of updates. You will be surprised by the amount of data that will show up in a short time period.
```